### ISSUE TITLE
Split registration from login

### ISSUE THREAD

**Issue by its-danny** [mist-id/mist#24]
Right now, there is no difference between the two. Because of this, we ask for user VCs every time they use the app, which isn’t the greatest experience. A way we *could* solve this is to split it into 2 steps:

1. The user scans a QR code, and we verify their identity.
2. The user scans a QR code and sends us their VCs.

Since we don’t create the user until the second is done, we’d check for the user after step 1 to decide whether or not they need to do step 2.

This is also not the best user experience, though, so I’d like to think more about it before moving forward with this.
